modules/github/widget.go

```diff
@@ -87,6 +129,14 @@ func (widget *Widget) currentGithubRepo() *GithubRepo {
 	return widget.GithubRepos[widget.Idx]
 }
 
+func (widget *Widget) openPr() {
+	currentSelection := widget.View.GetHighlights()
+	if widget.Selected >= 0 && currentSelection[0] != "" {
+		url := (*widget.currentGithubRepo().RemoteRepo.HTMLURL + "/pull/" +  widget.View.GetRegionText(currentSelection[0]))
+		utils.OpenFile(url)
```

Comment:
This is currently breaking on my install because the selected text is padded with empty spaces, so the URL ends up being `https://github.com/wtfutil/wtf/pull/%20472`. It would be better to use the selected index to get the repo from the widget's `GithubRepos` and then use the repo's `Number` attribute. That'll make this resistant to future styling changes.